src/content/protocols/catinabox.md

```diff
@@ -0,0 +1,122 @@
+| CIAB v1.1      | grantRole          | Grant any role to any address                                                      | Undeclared Multisig       |
+| CIAB v1.1      | revokeRole         | Remove any role any address                                                        | Undeclared Multisig       |
+| CIAB v1.1      | setFeeSplitAddress | Set the receiver of the protocol fee                                               | Undeclared Multisig       |
+| CIAB v1.1      | setIncentiveModule | Set the receiver of the protocol fee used to incentivize liquidity                 | Undeclared Multisig       |
+| CIAB v1.1      | setpsm             | Set the address of the peg stability module                                        | Undeclared Multisig       |
+| BoxETH         | mint               | mint boxETH token                                                                  | CIAB v1 and v1.1          |
+| BoxETH         | grantRole          | Grant any role to any address                                                      | CIAB v1 and v1.1 (can't make use of it) |
+| BoxETH         | revokeRole         | Remove any role any address                                                        | CIAB v1 and v1.1 (can't make use of it) |
+| PSM            | grantRole          | Grant any role to any address                                                      | Undeclared Multisig       |
+| PSM            | revokeRole         | Remove any role any address                                                        | Undeclared Multisig       |
+| PSM            | trade              | Exchange 1:1 BoxETH for stETH in the PSM                                           | Undeclared Multisig       |
+| PSM            | synthToFeeToken    | Convert the BoxETH in the PSM for BoxFEE and send it to the treasury               | Undeclared Multisig       |
+| PSM            | setTreasury        | Sets the address of the treasury                                                   | Undeclared Multisig       |
+| PSM            | setFeeToken        | Sets the address of the contract who receive BoxETH when synthToFeeToken is called | Undeclared Multisig       |
+| boxFEE staking | grantRole          | Grant any role to any address                                                      | Undeclared EOA            |
+| boxFEE staking | revokeRole         | Remove any role any address                                                        | Undeclared EOA            |
+| boxFEE staking | trade              | Exchange 1:1 BoxETH for stETH in the staking module                                | Undeclared Multisig & EOA |
+
+## Dependencies
+
+Fund for liquidity incentives are sent straight to the team treasury and any incentive campaign is done by hand.
+
+## Exit Window
+
+No timelocks have been found protecting the various permissioned functions in the protocol. All updates take effect immediately and potentialy retroactively for fee change.
```

Comment:
No timelocks have been found protecting the various permissioned functions in the protocol. All updates take effect immediately and potentially retroactively for fee change.